content/en/docs/collector/getting-started.md

```diff
@@ -159,14 +159,9 @@ $ sudo journalctl -u otel-collector
 
 ### Windows Packaging
 
-Every Collector release includes EXE and MSI packaging for Windows amd64 systems.
-The MSI packaging includes a default configuration that can be found at
-`\Program Files\OpenTelemetry Collector\config.yaml`.
+Windows releases are packaged as gzipped tarballs (`.tar.gz`) and will need to be unpacked with a tool that supports this compression format.
```

Comment:
I'd suggest making "releases" a link to https://github.com/open-telemetry/opentelemetry-collector-releases/releases